src/Pages/login.tsx

```diff
@@ -1,23 +1,44 @@
-import { ChangeEvent, useState } from "react";
-import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import { ChangeEvent, FormEvent, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
 
 const Login = () => {
-    const [username, setUserName] = useState<string>('');
+    const [email, setEmail] = useState<string>('');
     const [password, setPassword] = useState<string>('');
+    const [incorrectRes, setIncorrectRes] = useState<boolean>(false);
 
     const navigate = useNavigate();
+    const location = useLocation();
 
+    const logIn = (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault();    
+        axios.post('http://restapi.adequateshop.com/api/authaccount/login', {
+            email: email,
+            password: password
+        })
+            .then(function (response) {
```

Comment:
@ShiranthaKellum, it's better to add types for the `response` and use the ES6 arrow function instead of ES5 function,

example, 
```
// ES5
var x = function(x, y) {
   return x * y;
}

// ES6
const x = (x, y) => x * y;
```

refer: https://www.w3schools.com/js/js_es6.asp#mark_arrow